nextflow.config

```diff
@@ -248,6 +248,13 @@ profiles {
         shifter.enabled        = false
         charliecloud.enabled   = false
     }
+    wave {
```

Comment:
Wonder if this should be called `seqera-container`

Reply:
What about `community-containers`?

Reply:
Maybe you are right, though, because "Bio(containers)" is prefixed by the Bioconda community.